packages/bun-uws/src/Loop.h

```diff
@@ -58,12 +58,6 @@ struct Loop {
         for (auto &p : loopData->postHandlers) {
             p.second((Loop *) loop);
         }
-
```

Comment:
Mental notes: another solution is uncorking here but is unlikely to be a problem because we always use `HttpResponse *cork(MoveOnlyFunction<void()> &&handler)` on HttpResponse and `void cork(MoveOnlyFunction<void()> &&handler)` on WebSocket.

`this.global.bunVM().waitForPromise(promise);` can trigger this condition.